recipes/u3d/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: u3d
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/ClinicalGraphics/u3d.git
+  git_tag: {{ version }}
```

Comment:
Yeah, I think something changed about activation in `conda` (actually sounds like a bug fix). However, as we don't like cloning, I'm not sure whether we should actually have a fix in our docker image or whether we should just use this as a way of indicating to people that they need to fix things on there end. In any event, when `git` is actually needed, it can always be added as a `build` dependency.